pypit/arflux.py

```diff
@@ -87,7 +87,7 @@ def bspline_magfit(wave, flux, var, flux_std, nointerp=False, **kwargs):
     -------
     """
     from pypit import arutils
-    invvar = (var > 0.)/(var + (var <= 0.))
+    invvar = (var > 0.)/(np.max(var,0))
```

Comment:
It was X's idea :P

I'd be interested to sit in on any discussion about this /generating the sensitivity function, if it happens!